cypress/support/commands.js

```diff
@@ -22,4 +26,45 @@
 //
 //
 // -- This will overwrite an existing command --
-// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
\ No newline at end of file
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+const workerDate = {
+  firstName: faker.person.firstName(),
+  lastName: faker.person.lastName(),
+  email: faker.internet.email(),
+  age: Math.floor(Math.random() * (65 - 18 + 1)) + 18,
+  salary: Math.floor(Math.random() * (10000 - 100 + 1)) + 100,
```

Comment:
Magic numbers in the code. It's better to replace 10000 and 100 with meaningful constants.